app/move/views/confirmation.njk

```diff
@@ -0,0 +1,49 @@
+{% extends "layouts/base.njk" %}
+
+{% block customGtagConfig %}
+  gtag('set', {'page_title': 'Move completed'});
+{% endblock %}
+
+{% block pageTitle %}
+  {{ t("moves::confirmation.page_title", { name: fullname | upper }) }}
+{% endblock %}
+
+{% block beforeContent %}
```

Comment:
As discussed earlier, maybe we should override this block for this template as it's not that useful and creates two links to the same destination.